### DISCUSSION TITLE
SST to Wilcox 1998

### DISCUSSION THREAD

**Discussion by aeroamit** [su2code/SU2#1455] [General]
Hi,

I wanted to try and make required changes in SST model to convert it into Wilcox 1998 / 2006 models (https://turbmodels.larc.nasa.gov/wilcox.html). 
Which files, I should be looking into (including the files mentioned below and corresponding .hpp, .inl) -
SU2_CFD/src/variables/CTurbSSTVariable.cpp
SU2_CFD/src/solvers/CTurbSSTSolver.cpp
SU2_CFD/src/numerics/turbulent/turb_diffusion.cpp
SU2_CFD/src/numerics/turbulent/turb_sources.cpp
SU2_CFD/src/numerics/turbulent/turb_convection.cpp

Regards
Amit

**Comment by TobiKattmann** [comment]
Yep, correct. As of lately `turb_diffusion.cpp` and `turb_convection.cpp` do not exist anymore and their content moved to the hpp with the same name. 

Feel free to open a Draft PR whenever you have some things coded up :)

**Reply by aeroamit** [reply]
Thanks...
Will get back for the help...

**Reply by pcarruscag** [reply]
Hi @aeroamit dont forget we have developer meetings on Wednesdays at 3pm UK https://meet.jit.si/SU2_DevMeeting, if you want to talk about this "live".
Keep in mind that the way we have been using to add turbulence models is not very scalable and it is being re-considered in some of the open issues and PR's.

**Reply by aeroamit** [reply]
Thanks @pcarruscag ,
It will be good (for me) to join the meeting next week with some work done for the same on my end.
